src/__tests__/SignUp/SignUpScreen.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { shallow } from 'enzyme/'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+import SignUpScreen from '../../main/SignUp/SignUpScreen'
+import { createUser } from '../../main/SignUp/createUserApi'
+
+jest.mock('../../main/SignUp/createUserApi')
+jest.mock('react-redux')
+jest.mock('react-router-dom', () => ({
+  useHistory: jest.fn(),
+}))
+
+describe('SignUpScreen', () => {
+  let wrapper
+  beforeEach(() => {
+    jest.clearAllMocks()
+    wrapper = shallow(<SignUpScreen />)
+  })
+
+  it('render header with correct text', () => {
+    expect(wrapper.find({ 'data-testid': 'header' }).text()).toEqual('Sign Up')
+  })
+  describe('errorMessage', () => {
+    describe('when errorMessage does NOT exist', () => {
+      it('should not render errorMessage', () => {
+        const errorMessage = ''
+        useSelector.mockReturnValueOnce(errorMessage)
+        wrapper = shallow(<SignUpScreen />)
+        expect(wrapper.find({ 'data-testid': 'errorMessage' }).text()).toEqual(
+          '',
+        )
+      })
```

Comment:
we should make sure that if `useSelector` returns null, errorMessage does not render (aka, length = 0)

Reply:
fixed